targets/lenovo-x1-carbon.nix

```diff
@@ -99,6 +99,8 @@
               SUBSYSTEM=="input",ATTRS{name}=="ELAN067C:00 04F3:31F9 Touchpad",KERNEL=="event*",GROUP="kvm",SYMLINK+="touchpad"
               # Laptop TrackPoint
               SUBSYSTEM=="input",ATTRS{name}=="TPPS/2 Elan TrackPoint",GROUP="kvm"
+              # Lenovo X1 integrated webcam
+              SUBSYSTEM=="usb", ATTR{idVendor}=="04f2", ATTR{idProduct}=="b751", GROUP+="kvm"
```

Comment:
True, tested and works without the "+".
I'll create a separate patch for that since it is related to the webcam rather than pulseaudio.
I have to test the usb passthrough a bit more anyway.
Usb passthrough "works" but qemu usb device passthrough breaks after a while when the device has too much IO.